recipes/npcap/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from os.path import join
+from conan import ConanFile
+from conan.tools.files import get, download, copy
+
+
+class NpcapConan(ConanFile):
+    name = "npcap"
+    description = "Windows port of the libpcap library"
+    homepage = "https://npcap.com/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "https://github.com/nmap/npcap/blob/master/LICENSE"
```

Comment:
```suggestion
    license = ""LicenseRef-NPCAP"
```
https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/#d3-simple-license-expressions